src/Masa.Blazor.sln

```diff
@@ -45,52 +45,110 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "3 - CLI", "3 - CLI", "{AAE4
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Masa.Blazor.Doc.WebAssembly.Host", "Doc\Masa.Blazor.Doc.WebAssembly.Host\Masa.Blazor.Doc.WebAssembly.Host.csproj", "{64B28E6C-9AE0-48F1-B988-51FC7D314834}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "5 - Samples", "5 - Samples", "{909D8171-6A06-48CB-8F3B-E9629A42925C}"
```

Comment:
The samples directory  should be placed in the root directory and The sample project can be created as a separate solution

Reply:
What's your opinion? @doddgu

Reply:
Point 3: what do you think? what are your suggestions @doddgu

Reply:
I agree, it will be clearer.

Reply:
@whuanle Your folder name should be renamed Masa.Maui.Windows. Masa.Maui=>Masa.Maui.Windows